azure-quantum/README.md

```diff
@@ -1,18 +1,108 @@
-# Python SDK for Azure Quantum #
+![Azure Quantum logo](Azure-Quantum-logo.png)
```

Comment:
Will this logo need to be included in the package for PyPI's Markdown renderer to pick it up?

Reply:
Yes, see comment above. We also need to add a direct link to the file. Thanks for your review!

Reply:
Got it, I hadn't seen that when I posted, sorry.